clear/main.py

```diff
@@ -550,6 +551,25 @@ def route(prefix, vrf, namespace):
 helper = util_base.UtilHelper()
 helper.load_and_register_plugins(plugins, cli)
 
+@click.option('--all', '-a', is_flag=True, help='Delete also compressed logs')
+@cli.command()
+def logging(all):
+    """Clear logging files"""
+    if os.path.exists("/var/log.tmpfs"):
+        log_path = "/var/log.tmpfs"
+    else:
+        log_path = "/var/log"
+    
+    if all:
+        files_to_delete = glob.glob("{}/syslog*".format(log_path))
+    else:
+        files_to_delete = glob.glob("{}/syslog".format(log_path))
+    if os.path.isfile("{}/syslog.1".format(log_path))
+        files_to_delete += glob.glob("{}/syslog.1".format(log_path))
+
+    for f in files_to_delete:
+        cmd = ['sudo', 'rm' '-f',f]
```

Comment:
Rather than deleting files using the shell, would it be better to use the built-in python `os.remove(f)`? There might be permission issues, but it is better practice rather than calling the shell.

Reply:
doesn't work due to privileges, couldn't find a way to run os.remove with sudo/root privileges.